docs/Remove-PSReadlineKeyHandler.md

```diff
@@ -0,0 +1,107 @@
+---
+external help file: Microsoft.PowerShell.PSReadLine.dll-Help.xml
+keywords: powershell,cmdlet
+locale: en-us
+Module Name: PSReadLine
+ms.date: 12/07/2018
+online version: http://go.microsoft.com/fwlink/?LinkId=821451
+schema: 2.0.0
+title: Remove-PSReadLineKeyHandler
+---
+
+# Remove-PSReadLineKeyHandler
+
+## SYNOPSIS
+Removes a key binding.
+
+## SYNTAX
+
+```
+Remove-PSReadLineKeyHandler [-Chord] <String[]> [-ViMode <ViMode>] [<CommonParameters>]
+```
+
+## DESCRIPTION
+
+The `Remove-PSReadLineKeyHandler` cmdlet removes a specified key binding.
+
+## EXAMPLES
+
+### Example 1: Remove a binding
+
+```powershell
+Remove-PSReadLineKeyHandler -Chord Ctrl+Shift+B
```

Comment:
While V1 required `Shift`, V2 does not for normal keys like `B` or even `!`. In V1, you might have used `Ctrl+Shift+1` or `Ctrl+Shift+b`, in V2 you would use `Ctrl+!` or `Ctrl+B`. Also note that `Ctrl+b` is not the same as `Ctrl+B`.

This is worth noting because it is a minor breaking change.

Reply:
Will update.